Cargo.toml

```diff
@@ -13,9 +13,7 @@ include = [
 	"Cargo.lock",
 	"README.md",
 	"src/*",
-	"cuteconfig.default.toml",
-	"SUMMARY.default.toml",
-	"introduction.default.md",
+  "defaults/*",
 	"assets/*"
 ]
 
```

Comment:
Could you add the `.idea` directory to the `exclude` field?

Reply:
ye